logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/e2ei/usecase/EnrollE2EIUseCase.kt

```diff
@@ -182,9 +186,19 @@ class EnrollE2EIUseCaseImpl internal constructor(
                 return it.left()
             }
         } else {
+            // TODO secure
```

Comment:
can you be more clear what need to be secured here?

Reply:
I want to test it with E2EI which error should I propagate on this stage